modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -3,54 +3,60 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Studio enables you to import a RAML or OAS 2.0 and 3.0 API specification into either a new Mule project or an existing one.
+include::reuse::partial$beta-banner.adoc[tag="studio-asyncapi"]
 
-If you are using Mule runtime engine (Mule) 4.1.3 and earlier, or if you prefer to import an API specification that is not available in Exchange, follow the steps presented in xref:import-api-specification-local-file.adoc[import an API specification from a local file] or xref:import-api-specification-design-center.adoc[import an API specification from Design Center] depending on your preferred use case.
+Import an AsyncAPI, OAS, or RAML API spec into a new Mule project or an existing one.
 
-If you cannot access Exchange from your work environment, you can still import an API specification from your local Maven installation.
+If you're using Mule runtime engine (Mule) 4.1.3 and earlier, or if you prefer to import an API specification that isn't available in Exchange, you can xref:import-api-specification-local-file.adoc[] or xref:import-api-specification-design-center.adoc[].
```

Comment:
Does this sentence look right when built? Suggest either adding lowercase link text or making these "See xxx" links.

Reply:
The sentece reads when built:
"If you’re using Mule runtime engine (Mule) 4.1.3 and earlier, or if you prefer to import an API specification that isn’t available in Exchange, you can Import an API Specification from a Local File or Import an API Specification from Design Center."
If I change it, it would be repetitive and verbiage:
"If you’re using Mule runtime engine (Mule) 4.1.3 and earlier, or if you prefer to import an API specification that isn’t available in Exchange, you can Import an API Specification from a Local File or Import an API Specification from Design Center. See [Import an API Specification from a Local File](https://docs.mulesoft.com/studio/latest/import-api-specification-local-file) and [Import an API Specification from Design Center](https://docs.mulesoft.com/studio/latest/import-api-specification-design-center)."

Reply:
The point I was trying to make is that "you can Import an API Specification from a Local File or Import an API Specification from Design Center" is not proper sentence capitalization. 

Either use a See or hard code lowercase link text.

Reply:
OK! The hard-coded text seems to be the best solution.
Thanks!